### PR TITLE
Update Flux state change functions

### LINKED ISSUE #873: Document updated and new state changes functions
The `monitor.stateChanges` function has been updated and a new `monitor.stateChangesOnly` function has been added.

PR: https://github.com/influxdata/flux/pull/2675

##### Relevant URLs
- https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/monitor/statechanges/
- https://v2.docs.influxdata.com/v2.0/query-data/flux/monitor-states/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #873

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: monitor.stateChangesOnly() function
+description: >
+  The `monitor.stateChangesOnly()` function  takes a stream of tables that contains a` _level`
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md]: extra space after "function"

File: content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  column and returns a stream of tables where each record represents a state change.
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md]: change from `fromLevel` to `toLevel` only...or are there other state changes?

File: content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md (hunk 1 above)
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md]: "stream of tables" same as "stream of data"?

Comment (kelseiv) [reply content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md]: curious what is distinction between `monitor.stateChanges` and `monitor.stateChangesOnly`

File: content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## Examples
+
+##### Return records representing state changes
+```js
+import "influxdata/influxdb/monitor"
+
+
+monitor.from(start: -1h)
+  |> monitor.stateChangesOnly()
+```
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md]: maybe include sample output

File: content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md]: 1. Yes, they're the same. I updated them so they match.
2. `monitor.stateChanges` gives you to option of specifying which changes to detect (`any` to `crit`, `warn` to `info`, etc.). `monitor.stateChangesOnly` simply outputs rows where the state changes. It's actually used in the function definition for `monitor.stateChanges`.

File: content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md (hunk 2 above)
Comment (sanderson) [reply content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md]: stateChangesOnly just outputs whenever there's a change. It's not smart enough to know the levels.

File: content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md (hunk 3 above)
Comment (sanderson) [reply content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md]: Done

**Pull request merged**